### PR TITLE
Add line and line stop to plattform api

### PR DESCRIPTION
**Pull request by ozb-ioki**

### CODE REVIEW

File: lib/ioki/apis/platform_api.rb [hunk 1]
```diff
@@ -70,6 +70,17 @@ class PlatformApi
...
         model_class: Ioki::Model::Platform::Pause
       ),
+      Endpoints.crud_endpoints(
+        :line,
+        base_path:   [API_BASE_PATH, 'products', :id],
+        model_class: Ioki::Model::Platform::Line
+      ),
+      Endpoints.crud_endpoints(
+        :line_stop,
+        base_path:   [API_BASE_PATH, 'lines', :id],
```
Comment (tom-ioki) [inline lib/ioki/apis/platform_api.rb:80]: I guess `line_stops` are also part of the product resource. In that case you need to adjust the `base_path` to:
```ruby
base_path: [API_BASE_PATH, 'products', :id, 'lines', :id]
```

**Review by tom-ioki — CHANGES_REQUESTED** [review]
Great job so far, almost there.

Besides my comments, please have a look at the `spec/ioki/platform_api_spec.rb`. There you can add some specs for the added endpoints 👍

**Comment by ozb-ioki** [comment]
> Great job so far, almost there.
> 
> Besides my comments, please have a look at the `spec/ioki/platform_api_spec.rb`. There you can add some specs for the added endpoints 👍 

Thank you for the review Tom. I will look on it and apply the changes.

File: lib/ioki/apis/platform_api.rb (hunk 1 above)
Comment (ozb-ioki) [reply lib/ioki/apis/platform_api.rb:80]: Right, i had to push nested line_stops before, but changed it in the api after and did not apply here. Thanks.

**Review by tom-ioki — APPROVED** [review]
Nice! Looks good to me 🆗

**Pull request merged**